independent-publisher-connectors/ChuckNorris.io/README.md

```diff
@@ -0,0 +1,22 @@
+# Chuck Norris (ChuckNorris.IO)
+
+Chuck Norris facts are satirical factoids about martial artist and actor Chuck Norris that have become an Internet phenomenon and as a result have become widespread in popular culture. The 'facts' are normally absurd hyperbolic claims about Norris' toughness, attitude, virility, sophistication, and masculinity.
+
+Chuck Norris facts have spread around the world, leading not only to translated versions, but also spawning localized versions mentioning country-specific advertisements and other Internet phenomena. Allusions are also sometimes made to his use of roundhouse kicks to perform seemingly any task, his large amount of body hair with specific regard to his beard, and his role in the action television series Walker, Texas Ranger.
+
+[chucknorris.io](https://chucknorris.io) is a free JSON API for hand curated Chuck Norris facts.
+
+## Prerequisites
```

Comment:
@Laskewitz Please find a [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) for readme.md file and add the required sections.

Reply:
@sriyen-msft readme updated.